spec/registry.md

```diff
@@ -449,14 +457,19 @@ Number selection has three modes:
 - `ordinal` selection matches the operand to explicit numeric keys exactly
   followed by an ordinal rule category if there is no explicit match
 
+When the selection mode is `plural` implementations can optionaly support
```

Comment:
Why wouldn't `ordinal` work the same way?

Reply:
> Why wouldn't ordinal work the same way?

I think we discussed it in the last meeting and nobody was able to come up with a use case for ordinals.

But ICU supports it.
So I am open to add it.

Reply:
```
list = { Audrey, Bill, Carlos, David, Edna, Freda}

.local $first = {$list :get item=1}
.local $second = {$list :get item=2}
.local $user = {$list :get item=last}
.local $length = {$list :length}
.local $position = {$length :integer select=ordinal offset=2} 
.match $position
one {{After {$first} and {$second}, {$user} is {$position}st in line}}
two {{After {$first} and {$second}, {$user} is {$position}nd in line}}
... etc...
```

... but it's a pretty contrived use case...